### PR TITLE
MohammadH/ Refactor Instruments

### PR DESCRIPTION
**Pull request by mohammad-rework**
Changes:

-   Refactor all the instruments to only use `instruments_type` in `_symbol.js`

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [x] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Comment by niloo-fs** [comment]
Since we are converting the components to TS, I think it can be a good idea to convert this to ts too. instead of this everything is LGTM

File: src/pages/markets/components/helper/_symbol.js [hunk 1]
```diff
@@ -3,24 +3,14 @@ import PropTypes from 'prop-types'
...
-                    </SymbolContainer>
-                ))}
-            </React.Fragment>
-        ) : (
-            <SymbolContainer>
-                <img src={src} />
-                <Text>{text}</Text>
+        {instruments_type.map((symbol, index) => (
+            <SymbolContainer key={index}>
+                <img src={symbol.src} />
```
Comment (sean-binary) [inline src/pages/markets/components/helper/_symbol.js:10]:
```suggestion
                <img src={symbol.src} alt="" />
```
We can leave an empty alt here as a best practice for visual icons.

File: src/pages/markets/components/helper/_symbol.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+type SymbolProps = {
+    instruments_type: { src: string; text: ReactElement }[]
+}
+
+const Symbol = ({ instruments_type }: SymbolProps) => (
+    <React.Fragment>
+        {instruments_type.map((symbol, index) => (
+            <SymbolContainer key={index}>
+                <img src={symbol.src} alt="" />
```
Comment (niloo-fs) [inline src/pages/markets/components/helper/_symbol.tsx:13]: please add alt for this img tag

File: src/pages/markets/components/helper/_symbol.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import { Text } from 'components/elements'
+
+type SymbolProps = {
+    instruments_type: { src: string; text: ReactElement }[]
+}
+
+const Symbol = ({ instruments_type }: SymbolProps) => (
+    <React.Fragment>
+        {instruments_type.map((symbol, index) => (
+            <SymbolContainer key={index}>
```
Comment (niloo-fs) [inline src/pages/markets/components/helper/_symbol.tsx:12]: It's better not to use index as a key

Comment (mohammad-rework) [reply src/pages/markets/components/helper/_symbol.tsx:12]: Unfortunately we have no alternative to use instead of index.

**Review by sean-binary — APPROVED** [review]

**Pull request merged**